<ui version="4.0" >
 <class>trackListDlgUI</class>
 <widget class="QWidget" name="trackListDlgUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>KsCD - TrackList</string>
  </property>
  <property name="autoFillBackground" >
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="albumLbl" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>30</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt; font-weight:600;">Album&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QLabel" name="yearLbl" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>60</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-style:italic;">Year&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QTableWidget" name="trackTable" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>100</y>
     <width>561</width>
     <height>291</height>
    </rect>
   </property>
   <property name="lineWidth" >
    <number>1</number>
   </property>
   <property name="editTriggers" >
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors" >
    <bool>true</bool>
   </property>
   <property name="selectionMode" >
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior" >
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid" >
    <bool>false</bool>
   </property>
   <column>
    <property name="text" >
     <string>Track</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Title</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Length</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
